spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    name { "Hollaryem" }
+    username { "Yemyem" }
+    email { "yemyem@yahoo.com" }
+    password { "1234_@abcd" }
```

Comment:
Not super necessary, but I like using [faker](https://github.com/faker-ruby/faker) in factories to randomly generate different values for each field